### PR TITLE
Ruby: add 2.7.2 recipe

### PR DESCRIPTION
**Pull request by kenmays**
Ruby 2.7.2p137 built on hrev54672 x86-64

### CODE REVIEW

**Comment by extrowerk** [comment]
Build fails with baseruby.

File: dev-lang/ruby/ruby-2.7.2.recipe [hunk 1]
```diff
@@ -10,16 +10,16 @@ imperative. It also has a dynamic type system and automatic memory \
...
+COPYRIGHT="1993-2020 Yukihiro Matsumoto"
 LICENSE="Ruby
 	BSD (3-clause)"
-REVISION="3"
+REVISION="1"
 SOURCE_URI="https://cache.ruby-lang.org/pub/ruby/${portVersion%.*}/ruby-$portVersion.tar.xz"
-CHECKSUM_SHA256="27d350a52a02b53034ca0794efe518667d558f152656c2baaf08f3d0c8b02343"
-PATCHES="ruby-$portVersion.patchset"
+CHECKSUM_SHA256="1b95ab193cc8f5b5e59d2686cb3d5dcf1ddf2a86cb6950e0b4bdaae5040ec0d6"
+PATCHES="ruby-$portVersion.patchset"z
```
Comment (extrowerk) [inline dev-lang/ruby/ruby-2.7.2.recipe:19]: Typo.

**Comment by extrowerk** [comment]
Did you got any idea to fix the baseruby problem? If not, i think you can close this PR.

**Comment by kenmays** [comment]
x86-64 build works. I added -with-baseruby for x86 and it worked there (complained otherwise). I'll close this one as I notice unit test issues with current ruby 2.8.0-3.

**Comment by extrowerk** [comment]
Do i understand correctly you added --with-baseruby and also cmd:ruby to the BUILD_PREREQ? If so, that results a circular dependency. So you need ruby to build ruby, but you need ruby to build ruby. No, thats not a solution.

**Comment by kenmays** [comment]
> 
> 
> Do i understand correctly you added --with-baseruby and also cmd:ruby to the BUILD_PREREQ? If so, that results a circular dependency. So you need ruby to build ruby, but you need ruby to build ruby. No, thats not a solution.

No, you need ruby to test ruby... (was this done beforehand with no issues?)

**Comment by extrowerk** [comment]
Then check it please again.

kenmays <notifications@github.com>:
> > 
> > 
> > Do i understand correctly you added --with-baseruby and also cmd:ruby to the BUILD_PREREQ? 
> > If so, that results a circular dependency. So you need ruby to build ruby, but you need ruby 
to build ruby. No, thats not a solution.

**Pull request closed without merging**